docsrc/source/api/api_docs/modules/qat_config.rst

```diff
@@ -10,10 +10,7 @@ qat_config Module
 
 TrainingMethod
 ================================
-**Select a QAT training method:**
```

Comment:
there are many other docsrc modifications that need to be made for the upcoming release, I'm guessing this change is part of this PR because it was required to pass the build?

Reply:
Exactly. This workflow fails if the docs can not be build (like in this case...)